generator.js

```diff
@@ -30,21 +30,21 @@ module.exports = class WebpackGenerator extends Generator {
 					);
 				}
 
-				return this.prompt([Confirm('favicon', 'Do you want to add Favicon?')])
-					.then(answer => {
-						if (answer['favicon']) {
-							// TODO: Add the default value here
-							return this.prompt([Input('favPath', 'Enter your fav icon path :')]).then(answer => {
-								this.options.env.configuration.dev.topScope.push('const FaviconsWebpackPlugin = require("favicons-webpack-plugin");');
-								favPath = answer['favPath'];
-								this.options.env.configuration.dev.webpackOptions = createDevConfig(serviceWorker, favPath);
-								done();
-							});
-						} else {
-							this.options.env.configuration.dev.webpackOptions = createDevConfig(serviceWorker, favPath);
-							done();
-						}
-					});
+				return this.prompt([Confirm('favicon', 'Do you want to add Favicon?')]);
+			})
+			.then(answer => {
+				if (answer['favicon']) {
+					// TODO: Add the default value here
+					return this.prompt([Input('favPath', 'Enter your fav icon path :')]);
+				}
+			})
+			.then(answer => {
+				if (answer) {
+					favPath = answer['favicon'];
```

Comment:
this should be `favPath`

Reply:
Fixed.

Reply:
I'll add the tests by tomorrow. Can we merge this?